contrib/kubespray/example/master.csv

```diff
@@ -0,0 +1,3 @@
+openpai-master-01,10.1.0.1
+openpai-master-02,10.1.0.2
+openpai-master-03,10.1.0.3
```

Comment:
Remove multiple master, since we disallow apiserver ha

Reply:
Also precheck for multiple apiservers, fail it

---
In reply to: [365705237](https://github.com/microsoft/pai/pull/4096#discussion_r365705237) [](ancestors = 365705237)

Reply:
> Also precheck for multiple apiservers, fail it
> 
> In reply to: [365705237](https://github.com/microsoft/pai/pull/4096#discussion_r365705237) [](ancestors = 365705237)

It's environment pre-check. Will do it in another workitem.